README.md

```diff
@@ -1,250 +1,163 @@
-plugins, if required.
+# Authenticate against OpenShift
+$ oc login https://192.168.99.128:8443 -u developer -p developer
 
-### Reusing the Docker daemon
-
-When using a single VM of OpenShift its really handy to reuse the Docker daemon inside the VM; as this means you don't have to build on your host machine and push the image into a docker registry - you can just build inside the same docker daemon as minishift  which speeds up local experiments.
-
-To be able to work with the docker daemon on your mac/linux host use the [docker-env command](./docs/minishift_docker-env.md) in your shell:
-
-```
-eval $(minishift docker-env)
-```
-you should now be able to use docker on the command line on your host mac/linux machine talking to the docker daemon inside the minishift VM:
-```
-docker ps
-```
-
-## Managing your Cluster
-
-### Starting a Cluster
-
-The [minishift start](./docs/minishift_start.md) command can be used to start your cluster.
-This command creates and configures a virtual machine that runs a single-node Kubernetes cluster.
-This command also configures your [oc](http://kubernetes.io/docs/user-guide/kubectl-overview/) installation to communicate with this cluster.
-
-### Stopping a Cluster
-The [minishift stop](./docs/minishift_stop.md) command can be used to stop your cluster.
-This command shuts down the minishift virtual machine, but preserves all cluster state and data.
-Starting the cluster again will restore it to it's previous state.
+# Create Node.js example app
+$ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
 
-### Deleting a Cluster
-The [minishift delete](./docs/minishift_delete.md) command can be used to delete your cluster.
-This command shuts down and deletes the minishift virtual machine. No data or state is preserved.
+# Tail the built log until the app is build and deployed
+$ oc logs -f bc/nodejs-ex
 
-## Interacting With your Cluster
+# Expose a route to the service
+oc expose svc/nodejs-ex
 
-### OpenShift Client binary (oc)
+# Access the app
+$ curl http://nodejs-ex-myproject.192.168.99.128.xip.io
 
-The `minishift start` command creates a "[oc context](http://kubernetes.io/docs/user-guide/kubectl/kubectl_config_set-context/)" called "minishift".
-This context contains the configuration to communicate with your minishift cluster.
-
-Minishift sets this context to default automatically, but if you need to switch back to it in the future, run:
-
-```
-oc config set-context minishift
+$ minishift stop
+Stopping local OpenShift cluster...
+Stopping "minishift"...
 ```
+<a name="reusing-the-docker-daemon"></a>
+### Reusing the Docker daemon
 
-or pass the context on each command like:
-```
-oc get pods --context=minishift`
-```
+When running OpenShift in a single VM, it's recommended to reuse the Docker daemon Minishift uses
+for pure Docker use-cases as well.
+This means you don't have to build on your host machine and push the image into a docker registry.
```

Comment:
That's the same use case we advocate for ADB/CDK as well. This is about using the Docker daemon of this VM for all kind of Docker related purposes, not just OpenShift.